db/migrations/0002_actor_rename_first_name_genre_name_and_more.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.2 on 2023-09-07 20:04
+
+from django.db import migrations, models
```

Comment:
`0002_actor_rename_first_name_genre_name_and_more.py ` has too broad name. If you need to initially create models - do it only in initial migration. Try to create only one migration in this task :)